protocols/grpc/agent.proto

```diff
@@ -116,6 +117,87 @@ message UpdateContainerRequest {
 	LinuxResources resources = 2;
 }
 
+message StatsContainerRequest {
+    string container_id = 1;
+}
+
+message CpuUsage {
+	uint64 total_usage = 1;
```

Comment:
Presumably this is in bytes? Same question for lots of the other fields.

Reply:
checkout from libcontainer/cgroups/stats.go
To make it easier to transport into `kata event` display value, it should be in the same type with what `runc` does.